tests/unitTests/test.cpp

```diff
@@ -0,0 +1,186 @@
+/* Copyright (c) Stanford University, The Regents of the University of California, and others.
+ *
+ * All Rights Reserved.
+ *
+ * See Copyright-SimVascular.txt for additional details.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject
+ * to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS
+ * IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A
+ * PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER
+ * OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/* unit test for get_pk2cc
+*  this program generates a simple svFSI simulation to test function get_pk2cc.
+*  it allows you to create input deformation gradients to check the output 
+*  stress and tangent modulus. it usese Google Test for test cases.
+*/
+
+#include <stdlib.h>
+#include <iomanip>
+#include <iostream>
+#include "gtest/gtest.h"   // include GoogleTest
+#include "mat_fun.h"
+#include "mat_fun_carray.h"
+#include "mat_models.h"
+#include "mat_models_carray.h"
+#include "Simulation.h"
+#include "sv_struct.h"
+#include "distribute.h"
+#include "initialize.h"
+#include "read_files.h"
+
+
+// Using global variables: 
+// for the output: stress tensor and tangent tensor
+
+Array<double> S_0(3,3), Dm_0(6,6);      // for identity test
+Array<double> S_1(3,3), Dm_1(6,6);      // for non-identity test
+
+
+// Google Test test cases
+
+TEST(ExampleTest, OneEqualsToOne) {
+  EXPECT_EQ(1, 1);
+}
+
+TEST(GetPK2ccTest, Identity) {
+  // input: 
```

Comment:
Can you make three test cases with a different material each? Then it would already be a bit more modular for when we want to extend it